### PR TITLE
Add Kristina to web

### PR DESCRIPTION
**Pull request by kristinagalik**

### CODE REVIEW

File: people/kristina.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    "key": "kristina",
+    "name": "Kristína Gáliková",
+    "nick": "Kristína",
+    "email": "kristina.galikova@mild.blue",
+    "title_cs": "Full-stack developer",
+    "title_en": "Full-stack developer",
+    "title_de": "Full-stack developer",
+    "short_desc_cs": "Kristína študuje teoretickú informatiku na University of Glasgow. V Mild Blue sa venuje vývoju backendu a frontendu.",
+    "short_desc_en": "Kristina studies theoretical computing science at the University of Glasgow. In Mild Blue, she devotes her time to backend and frontend development.",
+    "short_desc_de": "Kristina studiert Theoretische Informatik an der University of Glasgow. In der Mild Blue entwickelt sie das Backend und Frontend."
```
Comment (tomaskourim) [inline people/kristina.json:11]: In Mild Blue entwickelt sie Backend sowohl auch Frontend von unseren Apps.

File: people/kristina.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+    "key": "kristina",
+    "name": "Kristína Gáliková",
+    "nick": "Kristína",
+    "email": "kristina.galikova@mild.blue",
+    "title_cs": "Full-stack developer",
+    "title_en": "Full-stack developer",
+    "title_de": "Full-stack developer",
+    "short_desc_cs": "Kristína študuje teoretickú informatiku na University of Glasgow. V Mild Blue sa venuje vývoju backendu a frontendu.",
+    "short_desc_en": "Kristina studies theoretical computing science at the University of Glasgow. In Mild Blue, she devotes her time to backend and frontend development.",
```
Comment (tomaskourim) [inline people/kristina.json:10]: she devotes her time to both backend and frontend development

**Review by tomaskourim — CHANGES_REQUESTED** [review]
takove pocitove jazykove upravy...

**Comment by kristinagalik** [comment]
@tomaskourim dakujem, opravene :)

**Comment by kubantjan** [comment]
Pak davej vzdycky prosim re-request review:
Screenshot from 2022-03-03 11-43-55

**Review by kubantjan — APPROVED** [review]
Za me dobre :)

**Pull request merged**